src/Grapevine.Tests/Server/PublicFolderFacts.cs

```diff
@@ -190,124 +267,325 @@ public void TrimsLeadingAndTrailingWhitespace()
             }
         }
 
-        public class RespondWithFileMethod
+        public class CreateDirectoryListKeyMethod
+        {
+            [Fact]
+            public void ReplacesBackslashWithForwardslash()
+            {
+                var folder = new PublicFolder();
+                var path = Path.Combine(folder.FolderPath, "path1", "path2");
+
+                path.Contains(@"\").ShouldBeTrue();
+                path.Contains(@"/").ShouldBeFalse();
+
+                path = folder.GetDirectoryListKey(path);
+
```

Comment:
Standardization. The `PathInfo` property will be using `/`, and the file system uses `\`, so I'm stanardizing on the `/` for when I do look ups.